core/src/test/java/org/apache/iceberg/TestFastAppend.java

```diff
@@ -489,32 +457,29 @@ public void testIncludedPartitionSummaries() {
         table.currentSnapshot().summary().keySet().stream()
             .filter(key -> key.startsWith(SnapshotSummary.CHANGED_PARTITION_PREFIX))
             .collect(Collectors.toSet());
-    Assert.assertEquals("Should include a partition summary", 1, partitionSummaryKeys.size());
+    assertThat(partitionSummaryKeys).hasSize(1);
 
     String summariesIncluded =
         table
             .currentSnapshot()
             .summary()
             .getOrDefault(SnapshotSummary.PARTITION_SUMMARY_PROP, "false");
-    Assert.assertEquals(
-        "Should set partition-summaries-included to true", "true", summariesIncluded);
+    assertThat(summariesIncluded).isEqualTo("true");
 
     String changedPartitions =
         table.currentSnapshot().summary().get(SnapshotSummary.CHANGED_PARTITION_COUNT_PROP);
-    Assert.assertEquals("Should set changed partition count", "1", changedPartitions);
+    assertThat(changedPartitions).isEqualTo("1");
 
     String partitionSummary =
         table
             .currentSnapshot()
             .summary()
             .get(SnapshotSummary.CHANGED_PARTITION_PREFIX + "data_bucket=0");
-    Assert.assertEquals(
-        "Summary should include 1 file with 1 record that is 10 bytes",
-        "added-data-files=1,added-records=1,added-files-size=10",
-        partitionSummary);
+    assertThat(partitionSummary)
```

Comment:
please also use containsEntry here. I don't think we need to define the `partitionSummary` variable a few lines above

Reply:
Sure, thanks for the advice.